src/screens/TopicPageScreen/index.js

```diff
@@ -32,11 +32,24 @@ import {getTopics, getUserTopic} from '../../service/topics';
 const styles = StyleSheet.create({
   parentContainer: {
     flex: 1
+  },
+  header: {
+    backgroundColor: 'lightblue',
+    justifyContent: 'center',
+    alignItems: 'center',
+    paddingTop: dimen.normalizeDimen(40),
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    right: 0,
+    zIndex: 2
+  },
+  headerText: {
+    fontSize: normalizeFontSizeByWidth(20),
+    fontWeight: 'bold'
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New styles have been added for the header. Ensure these changes are reflected correctly in the UI and do not break any existing layouts.
```diff
-   }
- });
-
- let lastDragY = 0;
+   },
+   header: {
+     backgroundColor: 'lightblue',
+     justifyContent: 'center',
+     alignItems: 'center',
+     paddingTop: dimen.normalizeDimen(40),
+     position: 'absolute',
+     top: 0,
+     left: 0,
+     right: 0,
+     zIndex: 2
+   },
+   headerText: {
+     fontSize: normalizeFontSizeByWidth(20),
+     fontWeight: 'bold'
+    }
+  });
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->